src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -87,7 +71,10 @@ public static void log(String command, String description, boolean success, WebD
     new Shooter().savePageScreenshot(screenPath + imageCounter, driver);
     CommonUtils.appendTextToFile(screenPath + imageCounter + ".html", getPageSource(driver));
     String className = success ? SUCCESS_CLASS : ERROR_CLASS;
-    String html = VelocityWrapper.fillLogRowWithScreenshot(Arrays.asList(className), command, description, imageCounter);
+    String
```

Comment:
Looks like you have broken style guides on your computer. Please follow steps to install: https://github.com/Wikia/guidelines/tree/master/Java